src/get-started/install/_get-sdk-linux.md

```diff
@@ -2,129 +2,98 @@
     ```
-    
- 1. Add the `flutter` tool to your path:
+
+3. Add the `flutter` tool to your path:
 
     ```terminal
     $ export PATH="$PATH:`pwd`/flutter/bin"
     ```
 
-    This command sets your `PATH` variable for the
-    _current_ terminal window only.
-    To permanently add Flutter to your path,
+    This command sets your `PATH` environment variable for the current
+    terminal window only.
+    To add Flutter as permenant part of your path,
     check out [Update your path][].
 
- 1. Optionally, pre-download development binaries:
-
-    The `flutter` tool downloads platform-specific
-    development binaries as needed. For scenarios
-    where pre-downloading these artifacts is preferable
-    (for example, in hermetic build environments,
-    or with intermittent network availability), iOS
-    and Android binaries can be downloaded ahead of time by running:
+4. (Optional) Pre-download development binaries:
 
     ```terminal
     $ flutter precache
     ```
 
-    For additional download options,
-    check out `flutter help precache`.
-
-You are now ready to run Flutter commands!
+    To find additional download options, run `flutter help precache`.
 
 {{site.alert.note}}
-  To update an existing version of Flutter,
-  check out [Upgrading Flutter][].
+  To update an existing version of Flutter, see [Upgrading Flutter][].
 {{site.alert.end}}
 
+### Post Installation: Run flutter doctor
 
-### Run flutter doctor
+After installing Flutter, run `flutter doctor`.
+This command checks your environment and displays a report in the
+terminal window.
+Flutter bundles the Dart SDK. You don't need to install Dart.
+Review the output for further tasks to perform.
 
-Run the following command to see if there are any
-dependencies you need to install to complete the setup
-(for verbose output, add the `-v` flag):
+For verbose output, add the `-v` flag:
 
 ```terminal
 $ flutter doctor
 ```
 
-This command checks your environment and displays
-a report to the terminal window.
-The Dart SDK is bundled with Flutter;
-it isn't necessary to install Dart separately.
-Check the output carefully for other software you might
-need to install or further tasks to perform (shown in **bold** text).
-
-For example:
+The output might resemble the following:
 
-<pre>
+```terminal
 [-] Android toolchain - develop for Android devices
     • Android SDK at /Users/dash/Library/Android/sdk
     <strong>✗ Android SDK is missing command line tools; download from https://goo.gl/XxQghQ</strong>
     • Try re-installing or updating your Android SDK,
       visit {{site.url}}/setup/#android-setup for detailed instructions.
-</pre>
-
-The following sections describe how to perform
-these tasks and finish the setup process.
+```
 
-Once you have installed any missing dependencies,
-run the `flutter doctor` command again to verify
-that you've set everything up correctly.
+After installing any missing dependencies,
```

Comment:
The previous text also discussed "tasks to perform", but the changes lose that. It would be good to still add that clarification since `flutter doctor` isn't just about dependencies.